### PR TITLE
User can view their entries

### PR DESCRIPTION
**Pull request by t-doan**
user-can-view-their-entries

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable string-quotes */
```
Comment (shawnkost) [inline css/styles.css]: Delete this. Don't add lint rules without asking. We have rules setup for a reason

File: js/main.js [hunk 2]
```diff
@@ -25,4 +35,71 @@ function handleSubmit(event) {
...
+  if (data.entries.length === 0) {
+    $h1.innerText = 'THERE ARE NO ENTRIES RECORDED';
+    $ul.appendChild($h1);
+  } else {
+    const existing = $ul.querySelector('h1');
+    if (existing) {
+      existing.remove();
+    }
+  }
+}
```
Comment (shawnkost) [inline js/main.js]: This text should be in your HTML file. The only purpose of this function is to check the length of `data.entries` and either show or hide the element accordingly. Not create it and delete it

File: js/main.js [hunk 3]
```diff
@@ -25,4 +35,71 @@ function handleSubmit(event) {
...
+
+function viewSwap(view) {
+  if (view === 'entries') {
+    document.getElementById('entries').classList.remove('hidden');
+    document.getElementById('entryForm').className = 'hidden';
+    data.view = view;
+  } else if (view === 'entryForm') {
+    document.getElementById('entryForm').classList.remove('hidden');
+    document.getElementById('entries').className = 'hidden';
```
Comment (shawnkost) [inline js/main.js]: You want to try and do your `querySelector`s globally for the most part. By doing them inside of a function, you're having to traverse the DOM tree over and over every time this function gets called. By moving this outside of the function, then you do the search once and always have a reference to it

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here Tai!

File: css/styles.css [hunk 4]
```diff
@@ -1,4 +1,3 @@
-/* stylelint-disable string-quotes */
```
Comment (t-doan) [inline css/styles.css]: made some changes.

File: index.html [hunk 5]
```diff
@@ -69,6 +69,7 @@ <h1>New Entry</h1>
           <h1>Entries</h1>
           <a href="#" class="entryForm"><h3 id="new-button">NEW</h3></a>
         </div>
+        <h1 id="noEntry" class="hidden">THERE ARE NO ENTRIES RECORDED</h1>
```
Comment (t-doan) [inline index.html:72]: add the no entries to html file

File: js/main.js [hunk 6]
```diff
@@ -6,6 +6,9 @@ const $formSubmit = document.querySelector('#entry-form');
 const $ul = document.querySelector('ul');
 const $entry = document.querySelector('.entry');
 const $entryForm = document.querySelector('.entryForm');
+const $noEntry = document.querySelector('#noEntry');
+const $entriesView = document.querySelector('#entries');
+const $entryFormView = document.querySelector('#entryForm');
```
Comment (t-doan) [inline js/main.js:11]: Created global variables to the functions

File: js/main.js [hunk 7]
```diff
@@ -80,26 +83,22 @@ function handleContent(event) {
...
   if (data.entries.length === 0) {
-    $h1.innerText = 'THERE ARE NO ENTRIES RECORDED';
-    $ul.appendChild($h1);
+    $noEntry.classList.remove('hidden');
   } else {
-    const existing = $ul.querySelector('h1');
-    if (existing) {
-      existing.remove();
-    }
+    $noEntry.setAttribute('class', 'hidden');
```
Comment (t-doan) [inline js/main.js]: Change to using a global variable and to only check for the data.entries.length and show/hide the message

File: js/main.js [hunk 8]
```diff
@@ -80,26 +83,22 @@ function handleContent(event) {
...
-    document.getElementById('entries').classList.remove('hidden');
-    document.getElementById('entryForm').className = 'hidden';
+    $entriesView.classList.remove('hidden');
+    $entryFormView.className = 'hidden';
     data.view = view;
   } else if (view === 'entryForm') {
-    document.getElementById('entryForm').classList.remove('hidden');
-    document.getElementById('entries').className = 'hidden';
+    $entryFormView.classList.remove('hidden');
+    $entriesView.className = 'hidden';
```
Comment (t-doan) [inline js/main.js]: Changed it all to use a global variable

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**